### PR TITLE
Fix naming tests

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: Emby.Naming/Common/NamingOptions.cs [hunk 1]
```diff
@@ -311,6 +311,12 @@ public NamingOptions()
                     }
                 },
 
+                // "Foo Bar 889"
+                new EpisodeExpression(@".*[\\\/](?![Ee]pisode)(?<seriesname>(\w+\s*?)*)\s(?<epnumber>\d{1,3})(-(?<endingepnumber>\d{2,3}))*[^\\\/]*$")
```
Comment (dkanada) [inline Emby.Naming/Common/NamingOptions.cs]: Is this a parsing fix or a new naming rule? Looks like the latter to me.

Comment (dkanada) [reply Emby.Naming/Common/NamingOptions.cs]: You should also move this below the comment about Kodi rules if this isn't directly from Kodi.

File: tests/Jellyfin.Naming.Tests/EpisodePathParserTest.cs [hunk 2]
```diff
@@ -31,24 +32,24 @@ public void ParseEpisodesCorrectly(string path, string name, int season, int epi
 
         [Theory]
         [InlineData("/media/Foo/Foo 889", "Foo", 889)]
-        [InlineData("/media/Foo/[Bar] Foo Baz - 11 [1080p]", "Foo Baz", 11)]
+        [InlineData("/media/Foo/[Bar] Foo Baz - 11 [1080p]", "Foo Baz", 11)]
```
Comment (dkanada) [inline tests/Jellyfin.Naming.Tests/EpisodePathParserTest.cs]: That would explain the issues. What character was used before?

Comment (Bond-009) [reply tests/Jellyfin.Naming.Tests/EpisodePathParserTest.cs]: No idea, haven’t checked yet.

File: Emby.Naming/Common/NamingOptions.cs (hunk 1 above)
Comment (Bond-009) [reply Emby.Naming/Common/NamingOptions.cs]: There is a test for this rule, so...
And one of the kodi rules gives a false positive, that’s why it’s before that one.

File: tests/Jellyfin.Naming.Tests/EpisodePathParserTest.cs (hunk 2 above)
Comment (dkanada) [reply tests/Jellyfin.Naming.Tests/EpisodePathParserTest.cs]: It was a no break space, whatever that means.

File: Emby.Naming/Common/NamingOptions.cs (hunk 1 above)
Comment (dkanada) [reply Emby.Naming/Common/NamingOptions.cs]: What would a false positive look like for a naming rule? Wouldn't that just be covered by the Kodi rule, or would it not always work consistently?

Comment (Bond-009) [reply Emby.Naming/Common/NamingOptions.cs]: It gave back 8 as season number, while there was none.

Comment (dkanada) [reply Emby.Naming/Common/NamingOptions.cs]: I would add that as a comment then, just in case this has further edge cases to account for in the future.

**Comment by joshuaboniface** [comment]
Backported into 10.4.2.

**Pull request merged**